Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/ui/details/RecoveryCertificateDetailsFragment.kt

```diff
@@ -73,6 +73,8 @@ class RecoveryCertificateDetailsFragment : Fragment(R.layout.fragment_recovery_c
     ) {
         qrCodeCard.bindValidityViews(certificate, isCertificateDetails = true)
         fullname.text = certificate.fullNameFormatted
+        icaoname.text = "${certificate.personIdentifier.lastNameStandardized}," +
+            " ${certificate.personIdentifier.firstNameStandardized}"
```

Comment:
should we create something like `certificate.fullNameStanardizedFormatted` ? 
Similar to `certificate.fullNameFormatted`